### PR TITLE
Enable STY Ads on Live

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**
- Remove the environment check, enabling STY page Ads on live

**Code changes:**

- Removes the env variable check for STY Ads

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
         "debug": "^4.1.0",
         "gensync": "^1.0.0-beta.1",
         "json5": "^2.1.2",
-        "lodash": "^4.17.20",
+        "lodash": "^4.17.19",
```
Comment (karinathomasbbc) [inline package-lock.json]: Is this intentional?

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/pages/StoryPage/StoryPage.jsx [hunk 2]
```diff
@@ -154,7 +154,6 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     path(['metadata', 'options', 'allowAdvertising'], pageData),
     adsEnabled,
     showAdsBasedOnLocation,
-    process.env.SIMORGH_APP_ENV !== 'live',
```
Comment (andrewscfc) [inline src/app/pages/StoryPage/StoryPage.jsx:157]: I'm pretty sure the tests for `StoryPage` will now fail as we have tests that check ads only work test but not live?

File: package-lock.json (hunk 1 above)
Comment (chris-hinds) [reply package-lock.json]: Nope, has been reverted now

File: src/app/pages/StoryPage/StoryPage.jsx (hunk 2 above)
Comment (chris-hinds) [reply src/app/pages/StoryPage/StoryPage.jsx:157]: What tests? They all passed locally

Comment (andrewscfc) [reply src/app/pages/StoryPage/StoryPage.jsx:157]: These tests all set the environment to be test: https://github.com/bbc/simorgh/blob/enable-sty-ads-on-live/src/app/pages/StoryPage/index.test.jsx#L421..L567 

I think that is no longer significant as the environment doesn't matter now is ads are enabled on all environments, I think the references to environments should be removed from these tests: https://github.com/bbc/simorgh/blob/enable-sty-ads-on-live/src/app/pages/StoryPage/index.test.jsx#L276..L567 

As it happens we never actually had a test to cover ads should not show on live like I thought we did 🙈

**Review by andrewscfc — APPROVED** [review]
Thanks Chris, the tests are a lot cleaner now 👍

File: src/app/pages/StoryPage/index.test.jsx [hunk 3]
```diff
@@ -274,15 +274,14 @@ describe('Story Page', () => {
...
   it.each`
-    environment | showAdsBasedOnLocation | showAdsBasedOnLocationExpectation
-    ${'live'}   | ${true}                | ${'permitted to be shown'}
-    ${'test'}   | ${true}                | ${'permitted to be shown'}
-    ${'live'}   | ${false}               | ${'not permitted to be shown'}
-    ${'test'}   | ${false}               | ${'not permitted to be shown'}
+    showAdsBasedOnLocation | showAdsBasedOnLocationExpectation
+    ${true}                | ${'permitted to be shown'}
+    ${true}                | ${'permitted to be shown'}
+    ${false}               | ${'not permitted to be shown'}
```
Comment (karinathomasbbc) [inline src/app/pages/StoryPage/index.test.jsx:280]: We can probably remove these duplicate lines now

Comment (chris-hinds) [reply src/app/pages/StoryPage/index.test.jsx:280]: Oops, just seen this. I'll do this in a separate PR

**Pull request merged**